packages/web3-common/src/eth_execution_api.ts

```diff
@@ -281,7 +286,7 @@ export type EthExecutionAPI = {
 	eth_sign: (address: Address, message: HexStringBytes) => HexString256Bytes;
 	eth_signTransaction: (
 		transaction: TransactionWithSender | Partial<TransactionWithSender>,
-	) => HexStringBytes;
+	) => HexStringBytes | SignedTransactionInfo;
```

Comment:
Geth seems to return this `SignedTransactionInfo` object, even though the ETH spec says it should be `HexStringBytes`